core/controlplane/config/templates/cluster.yaml

```diff
@@ -1177,6 +1177,8 @@ experimental:
   admission:
     podSecurityPolicy:
       enabled: false
+    alwaysPullImages:
```

Comment:
Would you mind adding a comment to suggest users to ensure that docker registries are reliable enough, and warn about otherwise the issue like @camilb [noted](https://github.com/kubernetes-incubator/kube-aws/pull/1009#discussion_r151197787) could happen?

What I'd expect would be:

```
Recommended to turn this on when and only when docker registries are reliable enough. Otherwise automatic pod restarts can be delayed due to temporally docker registry outages.
Please see https://github.com/kubernetes-incubator/kube-aws/pull/1009#discussion_r151197787 for more info.
```

// Please feel free to correct/improve any aspect of the sentences. I'm not a good writer 😄

Reply:
Sounds good to me. Added.